src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,40 @@
+package duke.task;
+
+public class Task {
+
+    private String description;
+    private boolean isDone;
+
+    protected static int taskCount = 0;
+
+    public Task(String desc) {
+        setDescription(desc);
+        setDone(false);
+        taskCount++;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "[X] " : "[ ] ");
+    }
+
+    public static int getTaskCount() {
+        return taskCount;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public void setDone(boolean isDone) {
+        this.isDone = isDone;
+    }
+
+    public String toText() {

```

Comment:
The names `toText` and `toString` are too similar. Since this one is for persisting data, maybe a better name is `toStorageString`